Documentation/dev-guide/interacting_v3.md

```diff
@@ -267,14 +314,29 @@ bar
 PUT
 foo
 bar_new
+```
 
+```bash
 # watch for changes on key `foo` since revision 3
 $ etcdctl watch --rev=3 foo
 PUT
 foo
 bar_new
 ```
 
+Here is an example to watch only from the last historical change:
+
+```bash
```

Comment:
```
PUT
 foo
 bar_new
 foo_latest
 bar_latest
```

Can you clarify what is foo, bar_new, foo_latest and bar_latest by adding inline comments? I am confused with the command `put foo_latest bar_latest` will trigger the watcher to return key `foo`